src/librbd/api/Mirror.h

```diff
@@ -101,7 +101,8 @@ struct Mirror {
   static int image_promote(ImageCtxT *ictx, bool force);
   static void image_promote(ImageCtxT *ictx, bool force, Context *on_finish);
   static void image_promote(ImageCtxT *ictx,
-                            const std::string &group_snap_id, bool force,
+                            const std::string &group_snap_id,
+                            bool m_group_snap_rollback, bool force,
```

Comment:
s/m_group_snap_rollback/group_snap_rollback

The prefix `m_` is used to denote that a variable is a class data member

Reply:
definitely unintentional, as you can see the .cc file doesn't mock it.